charts/pgadmin4/templates/secrets.yaml

```diff
@@ -12,4 +12,7 @@ data:
 {{- if .Values.serverDefinitions.enabled }}
   servers.json: {{ include "pgadmin.serverDefinitions" . | b64enc | quote }}
 {{- end }}
+{{- if .Values.env.ldapBindPassword }}
+  config_local.py: {{ include "env.ldapBindPassword" . | b64enc | quote }}
```

Comment:
Since this is a Python file, should the password to be assigned to some variable in its content, shouldn't it?

Reply:
I am closing this PR cause I don't know enough about it. Maybe in the future I can reopen it if I'll ever get LDAP experience.